test/morgan.js

```diff
@@ -822,6 +822,79 @@ describe('morgan()', function () {
         test.write('0')
       })
     })
+
+    describe('a custom', function () {
+      function token (req, res, arg) {
+        return arg
+      }
+      var url
+      beforeEach(function () {
+        url = morgan.url
+        morgan.token('url', token)
```

Comment:
Why even bother overwriting an existing token? Are you testing the ability to overwrite already-existing tokens? That seems like it should be a different test, as these tests seem to be trying to test a custom token function, in which case, defining a token with a new name would be a lot clearer in the below tests.